Userland/Libraries/LibRegex/RegexParser.cpp

```diff
@@ -1976,21 +1976,42 @@ Optional<ECMA262Parser::PropertyEscape> ECMA262Parser::read_unicode_property_esc
 {
     consume(TokenType::LeftCurly, Error::InvalidPattern);
 
-    auto start_token = m_parser_state.current_token;
-    size_t offset = 0;
-    while (match(TokenType::Char)) {
-        if (m_parser_state.current_token.value() == "}")
-            break;
-        offset += consume().value().length();
+    auto read_until = [&]<typename... Ts>(Ts && ... terminators)
```

Comment:
Style: Extraneous spaces around `&&` here, that looks like a boolean expression, not a type.

Reply:
Yeah clang-format doesn't like template lambdas yet. I'll turn it off here for now so it doesn't look so bad.

Reply:
clang-format and its silly edge cases :P